tasks/ca/statcan/geo.py

```diff
@@ -222,7 +250,10 @@ def columns(self):
 class Geography(TableTask):
 
     resolution = Parameter(default=GEO_PR)
-    year = IntParameter()
+    # FIXME: don't merge to master with this flag on. It's a workaround so
+    # CA can be run for the tiler without regenerating all the geographies.
+    # Once a new DO dump is deployed, this is not needed.
+    year = IntParameter(significant=False)
```

Comment:
We probably want to review the dump then, see what's missing and get it out of the way.

Reply:
I'm not sure about what you want to review. This change makes Tilesets API not having the latest geometries but the previous ones. Not a big deal. I'd remove this now, and rerun the XYZ tasks once we have a new DO dump with the latest CA geometries.